packages/quick_actions/quick_actions_ios/ios/quick_actions_ios.podspec

```diff
@@ -14,11 +14,14 @@ Downloaded by pub (not CocoaPods).
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/plugins/tree/main/packages/quick_actions' }
   s.documentation_url = 'https://pub.dev/packages/quick_actions'
-  s.source_files = 'Classes/**/*.{h,m}'
-  s.public_header_files = 'Classes/*.h'
-  s.private_header_files = 'Classes/PrivateHeaders/*.h'
+  s.source_files = 'Classes/**/*.{h,m,swift}'
+  s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
   s.platform = :ios, '9.0'
   s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
-  s.module_map = 'Classes/QuickActionsPlugin.modulemap'
+  s.swift_version = '5.0'
+  s.xcconfig = {
+    'LIBRARY_SEARCH_PATHS' => '$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)/ $(SDKROOT)/usr/lib/swift',
+    'LD_RUNPATH_SEARCH_PATHS' => '/usr/lib/swift',
```

Comment:
I hit similar issues in https://github.com/flutter/flutter/issues/16049#issuecomment-532476842.  But I don't think you'll need this once you add your first Swift file to the framework, right?   Instead of making this modulemap change now can you instead do it when you add the first Swift file so we don't need this frankly sketch workaround?

Reply:
Removed swift related updates since we don't have swift files yet.

Reply:
Instead of making this modulemap changes a standalone PR can you instead include it in the PR that actually adds a Swift file?  It doesn't seem worth a version bump, and I can't check it out and play with it with the Swift tests you write to see what else can be done instead.